### PR TITLE
Bug 1155088 - [Messages] Consider the receiver in message resend error case, r=azasypkin

### PR DESCRIPTION
**Pull request by steveck-chung**

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=44a22e39b5a810110e9895af0cc44e336bbfc68d

File: apps/sms/js/thread_ui.js [hunk 1]
```diff
@@ -2490,7 +2490,9 @@ var ThreadUI = {
         message: message,
         onerror: function onError(error) {
           var errorName = error.name;
-          this.showMessageSendingError(errorName);
+          // Only sms will have FDN error
+          var recipient = error.data.type === 'sms' && [error.data.receiver];
+          this.showMessageSendingError(errorName, {recipients: recipient});
```
Comment (azasypkin) [inline apps/sms/js/thread_ui.js]: Maybe it's just me, but it seems that if we skip message type checking it wouldn't hurt, but will make code simpler :
```
this.showMessageSendingError(
  error.name,
  { recipients: error.data.receivers || [error.data.receiver] }
);
```
I don't block on this, but what do you think?

Comment (steveck-chung) [inline apps/sms/js/thread_ui.js]: You're right that it won't hurt if we set the recipients even the error dialog don't need this. I just sync the behavior in send message but I'm fine to make this part simpler.

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=04573f2df8e432514eca96c1ede83a125f6676f1

**Comment by autolander** [comment]
http://docs.taskcluster.net/tools/task-graph-inspector/#sFZrw0QHTuCH2oNtFa-ilA

The pull request failed to pass integration tests. It could not be landed, please try again.

**Pull request merged**